core/shared/platform/common/posix/posix_clock.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2023 Amazon Inc.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include "platform_api_vmcore.h"
+#include "bh_platform.h"
+
+bool
+convert_clockid(bh_clock_id_t in, clockid_t *out)
```

Comment:
How about renaming to bh_clockid_to_clockid? And found that there is a different implementation in `core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c` with different function prototype?